samples/snippets/src/main/java/com/example/bigtable/HelloWorld.java

```diff
@@ -221,48 +217,32 @@ public List<Row> readTable() {
   }
 
   // [START bigtable_hw_create_filter]
-  public static void filterLimitCellsPerCol(String projectId, String instanceId, String tableId) {
+  public void filterLimitCellsPerCol(String tableId) {
```

Comment:
Should we make the following methods private as well?
- readTable
- readSpecificCells
- readSingleRow
- writeToTable
- close

Reply:
No, the public methods are being called by tests as well.
So, would prefer to keep it as such.